external/CMakeLists.txt

```diff
@@ -1 +1,2 @@
+add_definitions(-D_SILENCE_NONFLOATING_COMPLEX_DEPRECATION_WARNING)
```

Comment:
Why do you need this? Should it perhaps, instead, be a patch to LLVM? If it is already fixed upstream, can you pull in that patch as a backport?